archway/assetlist.json

```diff
@@ -43,6 +43,76 @@
       },
       "type_asset": "sdk.coin"
     },
+    {
+      "description": "BackBone Labs Liquid Staked Archway",
+      "type_asset": "cw20",
+      "address": "archway12yurzx8zynv3ck7uh4tucre48tqsm4fac4hfk9p3l24qs2cn08dqr684cg",
+      "denom_units": [
+        {
+          "denom": "archway12yurzx8zynv3ck7uh4tucre48tqsm4fac4hfk9p3l24qs2cn08dqr684cg",
+          "exponent": 0
+        },
+        {
+          "denom": "bARCH",
+          "exponent": 18
+        }
+      ],
+      "base": "archway12yurzx8zynv3ck7uh4tucre48tqsm4fac4hfk9p3l24qs2cn08dqr684cg",
+      "name": "BackBone Labs Liquid Staked Archway",
+      "display": "bARCH",
+      "symbol": "bARCH",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/archway/images/barch.png"
+      },
+      "traces": [
+        {
+          "type": "liquid-stake",
+          "counterparty": {
+            "chain_name": "archway",
+            "base_denom": "aarch"
+          },
+          "provider": "BackBoneLabs"
+        }
+      ],
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/archway/images/barch.png",
+          "theme": {
+            "primary_color_hex": "#040404"
+          }
+        }
+      ]
+    },
+    {
+      "description": "Architects Token",
+      "type_asset": "cw20",
+      "address": "archway16kxu278xtwa9ql20yhgk9smk2k6ke74hsxeesf36c9dhxqc5zyksh8ykwu",
+      "denom_units": [
+        {
+          "denom": "archway16kxu278xtwa9ql20yhgk9smk2k6ke74hsxeesf36c9dhxqc5zyksh8ykwu",
+          "exponent": 0
+        },
+        {
+          "denom": "TOKN",
+          "exponent": 18
+        }
+      ],
+      "base": "archway16kxu278xtwa9ql20yhgk9smk2k6ke74hsxeesf36c9dhxqc5zyksh8ykwu",
+      "name": "Architects Token",
+      "display": "TOKN",
+      "symbol": "TOKN",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/archway/images/tokn.png"
+      },
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/archway/images/tokn.png",
+          "theme": {
+            "primary_color_hex": "#5c84cc"
```

Comment:
How was a light blue color hex determined for this yellow-orange and black image?